integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala

```diff
@@ -172,10 +172,29 @@ object MojoImplementation {
       val analysisOut = None
       val sourceDirs = sourceDirs0.map(abs).toList
       val classesDir = abs(classesDir0)
+      val artifacts = project.getArtifacts().asScala
+      lazy val libraryAndDependencies = scalaContext.toList
+        .flatMap(_.findLibraryAndDependencies().asScala)
```

Comment:
In Scala 3, would this contain the `scala3-library`?

Reply:
in that case it will contain both `scala-library` and `scala3-library`